test/integration/targets/rds_param_group/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/aws
+posix/ci/cloud/aws
+skip/python3
```

Comment:
It would be good to fix the python 3 issues instead of skipping the tests on python 3.